Source/ColouringBook/ColouringBook.h

```diff
@@ -8,7 +8,7 @@
 //
 // Place any includes here that are needed by the majority of your .cpp files
 
-#include "EngineMinimal.h"
+#include "Engine.h"
```

Comment:
I think there are no debug logs being used in the PR'd version. Can we go back to EngineMinimal? I am not sure what the drawbacks of changing to the regular Engine.h are, so if we don't need it anymore, let's stay as it was.